index.js

```diff
@@ -82,3 +85,20 @@ function search() {
     })
   })
 }
+
+function addContact() {
+  $("#add-contact").on("click", function() {
+    $(".new-contact").fadeIn()
+  })
+}
+
+function create() {
+  $("#add").on("click", function() {
+    var newContact = $("#name-field").val()
+    contacts.push(newContact)
+    $(".contact-list").prepend(
```

Comment:
This `prepend` function looks very similar to the `listContacts` function. If you're adding the data to the contacts list, it _seems_ like you could just call it there and reuse the functionality.